wcomponents-lde/pom.xml

```diff
@@ -43,8 +43,15 @@
 			<artifactId>junit</artifactId>
 		</dependency>
 
```

Comment:
wcomponents-core has a "dummy" xslt for testing the "transform". What do you think of using something similar for this test and then it wont be dependant on the "theme"...